packages/core-typings/src/IMeteorError.ts

```diff
@@ -0,0 +1,22 @@
+export class MeteorError extends Error {
+	public isClientSafe = true;
+
+	public readonly errorType = 'Meteor.Error';
+
+	public constructor(public readonly error: string | number, public readonly reason?: string, public readonly details?: any) {
+		super(`${reason ? `${reason} ` : ''}[${String(error)}]`);
+	}
+
+	public toJSON(): any {
+		return {
+			isClientSafe: this.isClientSafe,
+			errorType: this.errorType,
+			error: this.error,
+			reason: this.reason,
+			message: this.message,
+			...(this.details && { details: this.details }),
+		};
+	}
+}
+
+export const isMeteorError = (error: any): error is MeteorError => error?.errorType === 'Meteor.Error';
```

Comment:
Are we using this?

Reply:
Not yet. Idea is to have another PR to move these types from core-services into core-typings, so i copied to whole file. I can remove as it's not used, won't affect. wdyt?

Reply:
I'd say it's better not to add unrelated stuff here. Thanks.